packages/browser/src/index.ts

```diff
@@ -75,6 +77,7 @@ export default class LogtoClient {
   constructor(logtoConfig: LogtoConfig, requester = createRequester()) {
     this.logtoConfig = {
       ...logtoConfig,
+      prompt: logtoConfig.prompt ?? PromptValue.Consent,
```

Comment:
?? => `||`

Reply:
since logtoConfig.prompt is the enum or undefined, keep using `??`